snap/info.go

```diff
@@ -32,6 +36,8 @@ type Info struct {
 	Apps        map[string]*AppInfo
 	Plugs       map[string]*PlugInfo
 	Slots       map[string]*SlotInfo
+	// info for store hosted snaps
+	Store *StoreInfo
```

Comment:
not in the rest api actually, only old snappy list and snappy info, so we could remove it or just call it StoreLastUpdated